src/huggingface_hub/repository.py

```diff
@@ -276,14 +373,15 @@ def clone_from(self, repo_url: str, use_auth_token: Union[bool, str, None] = Non
             # checks if repository is initialized in a empty repository or in one with files
             if len(os.listdir(self.local_dir)) == 0:
                 logger.warning(f"Cloning {clean_repo_url} into local empty directory.")
-                subprocess.run(
-                    ["git", "clone", repo_url, "."],
-                    stderr=subprocess.PIPE,
-                    stdout=subprocess.PIPE,
-                    check=True,
-                    encoding="utf-8",
-                    cwd=self.local_dir,
-                )
+                with lfs_log_progress():
+                    subprocess.run(
+                        f"git lfs clone {repo_url} .".split(),
```

Comment:
Doing the switch from `git clone` to `git lfs clone`. They're similar in terms of speed, but the smudge happens at different times.

Using `git clone` means that the process will not log any progress at first, before downloading all files - it will not log any "clean" operation. During at least half of the time, no progress will be shown to the user.

Using `git lfs clone`, the process will download the files before applying the clean filter. Both of these steps will be logged to the file, which can be shown to the user for feedback.